src/test/features/claim/routes/pay-by-account.ts

```diff
@@ -103,14 +107,30 @@ describe('Claim : Pay by Fee Account page', () => {
           .toLocation(ClaimPaths.claimSubmittedPage.uri.replace(':externalId', claimStoreServiceMock.sampleClaimObj.externalId)))
     })
 
+    it('should return 500 and render error page when form is valid and cannot retrieve case reference', async () => {
+      idamServiceMock.resolveRetrieveUserFor('1', ...roles)
+      draftStoreServiceMock.resolveUpdate()
+      feesServiceMock.resolveCalculateIssueFee()
+      claimStoreServiceMock.resolveRetrieveClaimByExternalId()
```

Comment:
That seems to be unneeded as retrieval should happen only when payment was successfully initiated. The same with `draftStoreServiceMock.resolveUpdate()` that should happen only when payment was successfully created.